src/docs/command-line.dox

```diff
@@ -260,7 +260,7 @@ Display the database log.
 The \c printlog command outputs the database log.
 
 @subsection util_printlog_synopsis Synopsis
-<code>wt [-rVv] [-C config] [-E secretkey ] [-h directory] printlog [-p] [-f output]</code>
+<code>wt [-rVv] [-C config] [-E secretkey ] [-h directory] printlog [-x] [-f output]</code>
```

Comment:
Not directly related to your change, but I just noticed that the `-rVv` should be `-RVv`.  Can you fix for all the places in there it is wrong please?